Description: this patch disables the cap on stevedore.
 Debian has python-stevedore 1.0.0 and it works fine.
 .
Origin: vendor
Forwarded: not-needed
Reviewed-By: gustavo panizzo <gfa@zumbi.com.ar>
Last-Update: <2014-12-17>

--- a/requirements.txt
+++ b/requirements.txt
@@ -17,4 +17,4 @@
 python-cinderclient>=1.1.0
 python-neutronclient>=2.3.6,<3
 requests>=2.2.0,!=2.4.0
-stevedore>=1.1.0  # Apache-2.0
+stevedore>=1.0.0  # Apache-2.0
